README.md

```diff
@@ -471,39 +471,40 @@ const pk: BBSPublicKey;
 // The signature
 const sig: BBSSignature = ...;
 
-// Prover prepares the attributes he wants to disclose, i.e. attribute index 2 and 4 (indexing is 0-based), and the ones he wants to hide. 
+// Prover prepares the attributes he wants to disclose, 
+// i.e. attribute index 2 and 4 (indexing is 0-based), and the ones he wants to hide. 
 const revealedMsgIndices: Set<number> = new Set();
 revealedMsgIndices.add(2);
-revealedMsgIndices.add(4);
+revealedMsgIndices.add(3);
```

Comment:
Why 4?

Reply:
I was trying to follow the messages array we constructed above which has only 4 messages. Now, by reviewing it, I see it's not suitable here, so I added a new messages array with 5 elements and changed the indexes accordingly. 

What do you think now?

Reply:
Thanks.